Sources/BraintreeCore/Analytics/FPTIBatchData.swift

```diff
@@ -37,6 +37,7 @@ struct FPTIBatchData: Codable {
         let payPalContextID: String?
         let timestamp: String
         let tenantName: String = "Braintree"
+        let venmoInstalled: Bool
```

Comment:
Looking at the tag in the tag directory of FPTI it looks like the expected underlying type here is a string.

Reply:
I think it gets converted to string in JSON.

Reply:
Adding a breakpoint in the `toDictionary` method shows that is is sent as a `Bool` to FPTI, but it seems like that is okay. We do the same for `is_simulator` which is also listed as a `String` type in the tag dictionary. As long as we are seeing this new tag as expected in FPTI we should be okay.